script/UI/HmGui/Components/Default/Container.lua

```diff
@@ -47,16 +64,40 @@ function Container:new(args)
         widthInLayout = args.widthInLayout,
         heightInLayout = args.heightInLayout,
         stackDirection = args.stackDirection or Enums.UI.StackDirection.Horizontal,
-        contents = args.contents
+        contents = args.contents,
+        color = {
+            background = args.color and args.color.background
+        },
+        showContainer = args.showContainer or function() return GameState.debug.metricsEnabled end,
+        showContainerColor = Color((math.random() + math.random(50, 99)) / 100, (math.random() + math.random(50, 99)) / 100, (math.random() + math.random(50, 99)) / 100, .4)
     }
 
     newContainer.render = function(self)
+        if not self.state.visible() then
+            return
+        end
+
+        -- color
+        if self.state.color().background then
+            Gui:setProperty(GuiProperties.BackgroundColor, self.state.color().background)
+        end
+
+        if self.state.showContainer() then
+            Gui:setProperty(GuiProperties.BorderColor, self.state.showContainerColor())
+        end
+
         if self.state.stackDirection() == Enums.UI.StackDirection.Horizontal then
```

Comment:
HmGui exposing now `GuiLayoutType` enum and `Gui:beginContainer(GuiLayoutType)` method.
You can use them now to make code simpler and avoid duplication of `GuiLayoutType`/`StackDirection` enums.